uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/manager/leave-requests-show-actions.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences-manager-leave-requests');
+var capturePath = './backstop_data/screenshots/reference/';
+
+// precondition: need to have the login of manager and all status types having atleast one leave request
+module.exports = function (casper) {
```

Comment:
Its not like we need to add test case for every requirement, the reason of having backstop js tests are to make sure the UI elements looks ok. But here we are testing if proper dropdown is present, which is already taken care by unit test. So I dont think its needed. Lets discuss with @AkA84